### PR TITLE
[DOCS-694] RUM new dashboards

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

It transforms the rum dashboard single page into a full section in order to document all OOTB dashboards available.

It also add some information on the main dashboard page in order to show the specific interaction with RUM OOTB dashboards.

### Motivation
It will be released soon

### Preview link

* https://docs-staging.datadoghq.com/gus/rum/real_user_monitoring/dashboards/

### Additional Notes

Screenshots and gifs are just placeholder here, we would need to wait for this feature to be in production in order to re-do them with the documentation data in it.

### CODE REVIEW

**Comment by hdelaby** [comment]
Actually, is it possible to change the image of the homepage to a GIF (https://docs-staging.datadoghq.com/gus/rum/real_user_monitoring/dashboards/) just to highlight the different clickable sections?

File: content/en/real_user_monitoring/dashboards/_index.md [hunk 1]
```diff
@@ -2,24 +2,49 @@
...
 
-{{< img src="real_user_monitoring/dashboards/rum_dashboard.png" alt="RUM Dashboard"  >}}
+When you [create a RUM application][1] dashboards are created within Datadog to analyze all the [data collected][2]. RUM dashboards can be found in the dashboards list and have the Datadog logo:
 
-When you [create a RUM application][1] a dashboard is created within Datadog to analyze all the [data collected][2].
-To access it, go in your [RUM application page][3] and select click on the **Dashboard** link associated with your application:
+{{< img src="real_user_monitoring/dashboards/rum_dashboard_in_dashlist.png" alt="RUM Dashboard in dash list" >}}
 
-{{< img src="real_user_monitoring/dashboards/application_page.png" alt="Application page"  >}}
+Another way to access them is to go in your [RUM application page][3] and click on the **Dashboard** links associated with your application:
```
Comment (cswatt) [inline content/en/real_user_monitoring/dashboards/_index.md]: You can also access these dashboards through your [RUM application page][3]. Click on the **Dashboard** links associated with your application:

File: content/en/real_user_monitoring/dashboards/errors_dashboard.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+further_reading:
+    - link: '/real_user_monitoring/installation/advanced_configuration'
+      tag: 'Documentation'
+      text: 'Advanced configuration for RUM data collection'
+    - link: '/real_user_monitoring/explorer'
+      tag: 'Documentation'
+      text: 'Explore your views within Datadog'
+---
+
+The errors dashboard provides insights about your applications’ errors. It is separated in 2 sections:
```
Comment (cswatt) [inline content/en/real_user_monitoring/dashboards/errors_dashboard.md]:
```suggestion
The errors dashboard provides insights about your applications’ errors. It is separated in two sections:
```

File: content/en/real_user_monitoring/dashboards/performance_overview_dashboard.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+further_reading:
+    - link: '/real_user_monitoring/installation/advanced_configuration'
+      tag: 'Documentation'
+      text: 'Advanced configuration for RUM data collection'
+    - link: '/real_user_monitoring/explorer'
+      tag: 'Documentation'
+      text: 'Explore your views within Datadog'
+---
+
+The performance overview dashboard offers a bird’s-eye view of RUM applications. It is separated in 3 sections:
```
Comment (cswatt) [inline content/en/real_user_monitoring/dashboards/performance_overview_dashboard.md]:
```suggestion
The performance overview dashboard offers a bird’s-eye view of RUM applications. It is separated in three sections:
```

File: content/en/real_user_monitoring/dashboards/performance_overview_dashboard.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+- **Performance metrics**:
+    For all views, 3 browser metrics are highlighted: First Contentful Paint, DOM Content Loaded, and Load Event. For each one of these metrics, widgets show the median, the 75th percentile, and the 90th percentile.
```
Comment (cswatt) [inline content/en/real_user_monitoring/dashboards/performance_overview_dashboard.md]:
```suggestion
    For all views, three browser metrics are highlighted: First Contentful Paint, DOM Content Loaded, and Load Event. For each one of these metrics, widgets show the median, the 75th percentile, and the 90th percentile.
```

File: content/en/real_user_monitoring/dashboards/performance_overview_dashboard.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- **Trends**:
+    Visualize the evolution of page views, front-end errors related to back-end calls failing, JS errors, and long tasks.
```
Comment (cswatt) [inline content/en/real_user_monitoring/dashboards/performance_overview_dashboard.md]:
```suggestion
    Visualize the evolution of page views, frontend errors related to backend calls failing, JS errors, and long tasks.
```

File: content/en/real_user_monitoring/dashboards/errors_dashboard.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- **General errors**:
+    Visualize which pages, browser, and countries are the most affected by front-end errors.
```
Comment (cswatt) [inline content/en/real_user_monitoring/dashboards/errors_dashboard.md]:
```suggestion
    Visualize which pages, browser, and countries are the most affected by frontend errors.
```

File: content/en/real_user_monitoring/dashboards/resources_dashboard.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+further_reading:
+    - link: '/real_user_monitoring/installation/advanced_configuration'
+      tag: 'Documentation'
+      text: 'Advanced configuration for RUM data collection'
+    - link: '/real_user_monitoring/explorer'
+      tag: 'Documentation'
+      text: 'Explore your views within Datadog'
+---
+
+The resources dashboard provides insights about your applications’ resources. It is separated in 4 sections:
```
Comment (cswatt) [inline content/en/real_user_monitoring/dashboards/resources_dashboard.md]:
```suggestion
The resources dashboard provides insights about your applications’ resources. It is separated in four sections:
```

**Pull request merged**